package.json

```diff
@@ -16,16 +17,26 @@
     "url": "https://github.com/fuse-box/fuse-react-demo/issues"
   },
   "homepage": "https://github.com/fuse-box/fuse-react-demo#readme",
-  "devDependencies": {
-    "fuse-box": "^3.5.0",
-    "node-sass": "^4.9.3",
-    "typescript": "^3.0.1",
-    "uglify": "^0.1.5",
-    "uglify-js": "^3.4.8"
-  },
   "dependencies": {
-    "fuse-react": "^1.0.19",
-    "react": "^16.4.2",
-    "react-dom": "^16.4.2"
+    "deep-equal": "1.0.1",
```

Comment:
why do we need all that?

Reply:
They were required with pnpm, which installs similar to yarn flat, just didnt think to take them out

Reply:
please remove them